### PR TITLE
Amazing super feature

### PR DESCRIPTION
**Pull request by GaPhil**
This PR removes old files that were unused and also adds a super cool new feature. A `Phone` class with increased functionality.

### CODE REVIEW

File: src/model/Phone.java [hunk 1]
```diff
@@ -6,9 +6,13 @@
 public class Phone {
 
     public int size;
+    public String model;
+    public int verion;
```
Comment (SimoneStefani) [inline src/model/Phone.java]: Here we have a typo: did you mean `version`?

File: src/model/Phone.java [hunk 2]
```diff
@@ -6,9 +6,13 @@
...
 
-    public Phone(int size) {
+    public Phone(int size, String model, int verion) {
         this.size = size;
+        this.model = model;
+        this.verion = verion;
```
Comment (SimoneStefani) [inline src/model/Phone.java]:
```suggestion
        this.version = version;
```

File: src/model/Phone.java [hunk 3]
```diff
@@ -18,4 +22,20 @@ public int getSize() {
...
+
+    public String getModel() {
+        return model;
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public int getVerion() {
```
Comment (SimoneStefani) [inline src/model/Phone.java]: Also here!

File: src/model/Phone.java [hunk 4]
```diff
@@ -18,4 +22,20 @@ public int getSize() {
...
+        return verion;
+    }
+
+    public void setVerion(int verion) {
```
Comment (SimoneStefani) [inline src/model/Phone.java]: Also here!

**Comment by GaPhil** [comment]
Please review the changes. Cheers!

**Pull request merged**